packages/core/src/integrations/requestdata.ts

```diff
@@ -166,15 +133,3 @@ function convertReqDataIntegrationOptsToAddReqDataOpts(
     },
   };
 }
-
-function getSDKName(client: Client): string | undefined {
```

Comment:
Why did we even do this before here 😅 but I guess it is just not needed because we always set this directly now, right?